srcpkgs/asm6/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'asm6'
+pkgname=asm6
+version=1.6
+revision=1
+create_wrksrc=yes
+hostmakedepends="unzip"
+short_desc="Simple but functional 6502 assembler"
+maintainer="rc-05 <rc23@email.it>"
+license="public domain"
+homepage="http://3dscapture/NES"
```

Comment:
How should I indicate that the package is of public domain?
I've tried _public domain_ and _Public Domain_ but neither of these are accepted.

Reply:
We have no consensus how to mark public domain, use anything and ignore warning. I meant typo in homepage.

Reply:
Sorry for the late reply: I've fixed the URL to point to the HTTPS site instead of the unencrypted version and by adding the .com at the end of the domain.

Honestly the first time I didn't catch that: thanks for that! :smiley: